pkgs/development/haskell-modules/initial-packages.nix

```diff
@@ -0,0 +1,2 @@
+args@{ pkgs, stdenv, callPackage }: self:
+  (import ./hackage-packages.nix args self) // (import ./hie-packages.nix args self)
```

Comment:
Please reverse the order of that set update to make sure that no entry in hie-packages can ever replace an entry of the same name in hackage-packages.

Reply:
👍